### ISSUE TITLE
Failure and permanent hang with yarn 2 pnp

### ISSUE THREAD

**Issue by bbugh** [johnsoncodehk/vue-tsc#45]
When running `yarn vue-tsc --noEmit` with Yarn 2 pnp, this is the result:

```
Content-Length: 76

{"seq":0,"type":"event","event":"typingsInstallerPid","body":{"pid":32571}}
```

It works fine with npm or yarn 1.x. 

This can be easily reproduced by cloning `https://github.com/bbugh/vue-tsx-tailwind-starter`, then adding `vue-tsc` to the `devDependencies` and trying to run it.

**Comment by johnsoncodehk** [comment]
Can you check if the situation is the same?

https://github.com/johnsoncodehk/volar/issues/283#issuecomment-881231493

**Comment by sto3psl** [comment]
I'm seeing the same issue with `npm@7.20.0`, `vue-tsc@0.2.1` and `typescript@4.3.5`.

It works with `vue-tsc@0.2.0` and `typescript@4.2.4` but as soon as I install `vue-tsc@0.2.1` the message appears again and it hangs.

**Comment by sto3psl** [comment]
Follow up: `vue-tsc@0.2.0` and `typescript@4.3.5` works, too.

I only get the problematic message and hang with `vue-tsc@0.2.1`.

**Comment by Shinigami92** [comment]
Can reproduce with yarn `1.22.10`, vue-tsc `0.2.1` and ts `4.3.5`

![image](https://user-images.githubusercontent.com/7195563/126123749-c6c213e7-202d-4fce-8a5c-d5d9fcff4635.png)

**Comment by trapcodeio** [comment]
I just encountered this error.
had to downgrade to `0.2.0` any fix yet?

**Comment by johnsoncodehk** [comment]
@sto3psl @Shinigami92 @trapcodeio thanks for reports, this is regression in vscode-vue-language@0.26.4 and fixed in https://github.com/johnsoncodehk/volar/commit/a6e3085835a86c6c7c8b61d1854c9ce68336a8c5.

**Comment by Shinigami92** [comment]
It's working 🎉

**Comment by johnsoncodehk** [comment]
> @sto3psl @Shinigami92 @trapcodeio thanks for reports, this is regression in vscode-vue-language@0.26.4 and fixed in [johnsoncodehk/volar@a6e3085](https://github.com/johnsoncodehk/volar/commit/a6e3085835a86c6c7c8b61d1854c9ce68336a8c5).

Released in vscode-vue-language@0.26.6.
